rules/docker-snmp-sv2.mk

```diff
@@ -3,7 +3,7 @@
 DOCKER_SNMP_SV2 = docker-snmp-sv2.gz
 $(DOCKER_SNMP_SV2)_PATH = $(DOCKERS_PATH)/docker-snmp-sv2
 ## TODO: remove LIBPY3_DEV if we can get pip3 directly
-$(DOCKER_SNMP_SV2)_DEPENDS += $(SNMP) $(SNMPD) $(PY3) $(LIBPY3_DEV)
+$(DOCKER_SNMP_SV2)_DEPENDS += $(SNMP) $(SNMPD) $(PY3) $(LIBPY3_DEV) $(SONIC_UTILS)
```

Comment:
Ultimately, there should be no need to install the sonic-utilities package in the SNMP docker. SNMP shouldn't rely on the CLI, but rather just the PSU plugin. However, the plugin bases currently live in the sonic-utilities repo. If we could move all of our Python modules into their own repo/Debian package (e.g., "sonic-python-modules"), we could simply install that package here, and wouldn't need to install all of the dependencies.

Reply:
@jleveque @lguohan 
From my point of view, the refactoring of sonic-utilities dependencies can be done separately as we need PSU feature available ASAP. 
I use the psuutil CLI to avoid the code duplication. In case to use plugin directly, we need to copy ("duplicate") the logic for loading vendor and platform specific  plugin to SNMP MIB implementation.

Reply:
I agree we need this ASAP. This was simply intended as a comment to consider for the future. I did not intend for you to make this change now.

Reply:
If there is plan to refactor in near future, please add a TODO comment as above.

Reply:
@andrii-savka: FYI, from inside the SNMP docker, you would not need to duplicate the logic for loading vendor and platform specific plugin, because each docker container mounts the appropriate platform and hwsku directories to `/usr/share/sonic/platform/` and `/usr/share/sonic/hwsku/`, respectively. Therefore, from inside the docker, you can simply load the plugin from `/usr/share/sonic/platform/plugins`.

Reply:
@jleveque @qiluo-msft 
The "TODO" mark is added. 
Regarding the plugin usage instead of CLI. 
Currently, we can't avoid we sonic-utilities installing to SNMP docker because psu_base.py which required by vendor specific plugin is placed in sonic-util. repo.

Reply:
Anyway, do we still need to use plugin directly from "/usr/share/sonic/platform/plugins/" instead of CLI in SNMP?

Reply:
One of the main idea is that SNMP agent know nothing about PSU implementation. It just used a CLI to get a PSU status with no matter whether PSU CLI is plugin, C app or some script.

Reply:
Agreed to use PSU plugin in SNMP with current dependencies.